assets/css/_route_ladder.scss

```diff
@@ -36,3 +36,30 @@
     fill: $charcoal-grey;
   }
 }
+
+.m-incoming-box {
```

Comment:
This name (`.m-incoming-box` instead of `m-route-ladder__incoming-box`) is a step towards making the incoming box its own module. What do you think about pulling it all the way out into its own `.tsx` and `.scss` files? It'd maintain the one-to-one-to-one relationship between `.tsx` files, `.scss` files, and css class names, and might make the reorganization in the asana task [Make all Incoming Boxes the same height](https://app.asana.com/0/1112935048846093/1125006647738997) easier.